kubernetes-client/package.yaml

```diff
@@ -0,0 +1,45 @@
+name: kubernetes-client
+version: 0.1.0.0
+description: |
+  This package contains functions for working with kubeconfig files.
+
+  Usage of kubeconfig files are described at https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
+library:
+  source-dirs: src
+tests:
+  spec:
+    main: Spec.hs
+    source-dirs: test
+    dependencies:
+      - kubernetes-client
+      - hspec
+      - yaml
+extra-source-files:
+  - test/testdata/*
+dependencies:
+  - base >=4.7 && <5.0
+  - bytestring >=0.10.0 && <0.11
+  - aeson >=1.0 && <2.0
+  - connection
+  - containers
+  - data-default-class
+  - http-client >=0.5 && <0.6
+  - http-client-tls
+  - kubernetes-openapi-client-gen == 0.1.0.0
```

Comment:
This should change to kubernetes-client-core right?

Reply:
I will update it when I regen the code to use the new package name.

Reply:
:+1: